modules/heat_conduction/doc/content/source/userobjects/GapFluxModelRadiation.md

```diff
@@ -4,8 +4,44 @@
 
 ## Description
 
-`GapFluxModelRadiation` computes a radiative heat flux across a gap following the existing implementation of radiation physics. It is used by
-[ModularGapConductanceConstraint.md](ModularGapConductanceConstraint.md).
+`GapFluxModelRadiation` computes a radiative heat flux across a gap following the
+diffusion approximation of radiation physics. This user object must be used in
+combination with [ModularGapConductanceConstraint](ModularGapConductanceConstraint.md).
+
+The heat flux across the gap is given by the classical expression
+\begin{equation}
+  \label{eqn:radiationHeatFlux}
+  q_r = \sigma F_e \left( T_s^4 - T_f^4 \right) \sim h_r \left(T_s - T_f \right)
+\end{equation}
+where $\sigma$ is the Stephan-Boltzmann constant, $F_e$ is an emissivity function,
+$T_s$ is the surface temperature, $T_f$ is the farfield temperature, and $h_r$
+is the radiant gap conductance. This expression can be rearranged to solve for $h_r$:
+\begin{equation}
+  h_r = \sigma F_e \frac{\left( T_s^4 - T_f^4 \right)}{\left( T_s - T_f \right)}
+\end{equation}
+which reduces to
+\begin{equation}
+  \label{eqn:radiantGapConductance}
+  h_r = \sigma F_e \left( T_s^2 + T_f^2 \right) \left( T_s + T_f \right).
+\end{equation}
+
+The emissivity function is given as the harmonic mean of the two emissivity values,
+approximating the gap as defined by two infinite parallel planes.
+\begin{equation}
+  \label{eqn:emissivityFunction}
+  F_e = \frac{1}{\left( 1/e_s - 1/e_f -1 \right)}
+\end{equation}
+where $e_s$ and $e_f$ are the near surface and farfield emissivity values, respectively.
```

Comment:
This isn't actually a harmonic mean is it? I think a harmonic mean would be 2 / (1/es + 1/ef). I'm not debating that this is the correct formulate for the emissivity (although we need the change the sign of -1/ef to +1/ef (it's correct in the source code)), just debating the description of it.

@sapitts I've seen this formula in random articles around the internet, but I don't suppose you have a bona fide reference that we could use here? I'm mostly asking for my own interest for hopefully learning more on emissivity factor computation in different geometries

Reply:
@recuero do you by chance know a good reference for emissivity formulas?

Reply:
@snschune @GiudGiud any suggestions on good references we can cite for these kinds of things (and for me to look at for my own knowledge? :smile: )

Reply:
@sapitts pointed me to https://en.wikipedia.org/wiki/Thermal_radiation#cite_note-ReferenceA-19 which looks like a good one. Maybe I can find it on the interwebs somewhere

Reply:
@lindsayad, not sure if you are interested in this but IIRC our radiation formulas are not accurate for non-plate geometries and large gaps (see #21025). Probably that reference that Stephanie pointed to has info on that.

Reply:
Thanks for linking to that issue! I'm trying to do some updates for mortar thermal contact for cylinders in https://github.com/idaholab/moose/pull/21776. I should reference that issue there

Reply:
if you look on page 347 of this pdf of the Todreas and Kazimi there's formulas about emissivities
https://ansccny.files.wordpress.com/2015/01/nuclear-1-thermal-hydraulic-fundamentals.pdf

Reply:
this is the old edition btw and it is full of mistakes, dont rely on it. You d want the new one

Reply:
Turns out the embarrassment of a typo never dulls with time, but I'm glad you caught the incorrect negative sign in this documentation @lindsayad . Do you have a pull request with the correction in the works? If not, I'll get one put together quickly.

Still no luck on unearthing my heat transfer textbook but I haven't given up on finding it

Reply:
@sapitts https://github.com/idaholab/moose/pull/21776 has the doc fix. I make typos in documentation all the time haha. More important is that it's right in the code ... which it is. You can hardly be penalized as there was no documentation before your contribution here!

Reply:
Thanks for sharing @GiudGiud. Yea that looks like the infinite parallel plate emissivity model on 347 (of the pdf, 333 page numbering)